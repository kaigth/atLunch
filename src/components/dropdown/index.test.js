import React from "react";
import { render } from '@testing-library/react';
import DropDown from '.';

describe('DropDown', () => {
    test('matches snapshot', () => {
        const { container } = render(<DropDown />)

        expect(container).toMatchSnapshot();
    });
});
