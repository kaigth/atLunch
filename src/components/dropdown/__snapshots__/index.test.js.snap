// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropDown matches snapshot 1`] = `
<div>
  <div
    class="dropdown__splash "
  />
  <div
    class="dropdown "
  >
    <div
      class="dropdown__content"
    />
    <div
      class="dropdown__footer"
    >
      <button
        class="button button--borderless"
      >
        Apply
      </button>
    </div>
  </div>
</div>
`;
